notebooks/GWAS_blood_pressure_p2/requirements.txt

```diff
@@ -1,3 +1,2 @@
 terra-notebook-utils
-herzog >= 0.0.2, < 0.1.0
-tenacity
\ No newline at end of file
+herzog >= 0.1.0, < 0.2.0
```

Comment:
I'm curious -- why do you limit the version? I get not wanting below 0.1.0, but is < 0.2.0 a method of futureproofing in case herzog changes?

Reply:
Yep, I'm using minor version changes to indicate a breaking change in Herzog's API